### PR TITLE
Fixes example snippets in CString.swift

### PR DESCRIPTION
**Pull request by valeriyvan**
Fixes example snippets in CString.swift

### CODE REVIEW

File: stdlib/public/core/CString.swift [hunk 1]
```diff
@@ -123,7 +123,7 @@ extension String {
   ///                                      repairingInvalidCodeUnits: true)
   ///         print(s)
   ///     }
-  ///     // Prints "Optional((Caf�, true))"
```
Comment (CodaFi) [inline stdlib/public/core/CString.swift:126]: This example appears to print 

```
Optional((result: "Caf�", repairsMade: true))
```

Comment (valeriyvan) [reply stdlib/public/core/CString.swift:126]: fixed

**Comment by CodaFi** [comment]
@swift-ci smoke test

**Comment by CodaFi** [comment]
⛵

**Pull request merged**